src/mono/nuget/Microsoft.NET.Runtime.WorkloadTesting.Internal/Sdk/WorkloadTesting.Core.targets

```
@@ -12,6 +12,9 @@
 
     <SdkWithWorkloadStampPath>$(SdkWithWorkloadForTestingPath)version-$(SdkVersionForWorkloadTesting).stamp</SdkWithWorkloadStampPath>
     <SdkWithWorkload_WorkloadStampPath>$(SdkWithWorkloadForTestingPath)workload.stamp</SdkWithWorkload_WorkloadStampPath>
+
+    <TemplateNuGetConfigPathForWorkloadTesting Condition="'$(TemplateNuGetConfigPathForWorkloadTesting)' == '' and '$(RepoRoot)' != ''">$(RepoRoot)NuGet.config</TemplateNuGetConfigPathForWorkloadTesting>
+    <RunInstallWorkloadUsingArtifactsAfterThisTarget Condition="'$(RunInstallWorkloadUsingArtifactsAfterThisTarget)' == ''">ArchiveTests</RunInstallWorkloadUsingArtifactsAfterThisTarget>
```

nit: `InstallWorkloadUsingArtifactsAfterTargets` instead of `RunInstallWorkloadUsingArtifactsAfterThisTarget`

```suggestion
    <InstallWorkloadUsingArtifactsAfterTargets Condition="'$(RunInstallWorkloadUsingArtifactsAfterThisTarget)' == ''">ArchiveTests</InstallWorkloadUsingArtifactsAfterTargets>
```

---

+ usage lower in the file